docs/swarm/swarm-tutorial/index.md

```diff
@@ -50,16 +50,26 @@ provider. This tutorial uses the following machine names:
 
 ###  Docker Engine 1.12 or later
 
-To use swarm mode, you must [install Docker Engine](../../installation/index.md)
-on each one of the host machines. Alternatively, install the latest Docker for
-Mac or Docker for Windows.
+This tutorial requires that you are running Docker Engine 1.12 or later on each of the host machines.
 
->**Note**: Docker for Mac and Docker for Windows let you use single-node
-features of swarm mode, like creating a swarm and creating a service. Multi-node
-features like joining additional nodes and scaling a service are not available.
+You can get Docker Engine v1.12 by installing it directory on Linux based systems, or with Docker for Mac or Docker for Windows.
```

Comment:
Thanks , I'll try to make it clearer that both platforms are discussed below in subtropics. Also I see some typos I'll fix.